src/nplinker/metabolomics/load_gnps.py

```diff
@@ -755,30 +414,10 @@ def load_clusterinfo_fbmn(strains, nodes_file, extra_nodes_file, md_table_file,
     return spec_info, unknown_strains
 
 
-def load_dataset(strains,
-                 mgf_file,
-                 edges_file,
-                 nodes_file,
-                 quant_table_file=None,
-                 metadata_table_file=None,
-                 ext_metadata_parsing=False):
-    # common steps to all formats of GNPS data:
-    #   - parse the MGF file to create a set of Spectrum objects
-    #   - parse the edges file and update the spectra with that data
-
-    # build a set of Spectrum objects by parsing the MGF file
-    ms1, ms2, metadata = LoadMGF(name_field='scans').load_spectra([mgf_file])
-    logger.info(f'{len(ms1)} molecules parsed from MGF file')
-    spectra = mols_to_spectra(ms2, metadata)
-    # above returns a list, create a dict indexed by spectrum_id to make
-    # the rest of the parsing a bit simpler
-    spec_dict = {spec.spectrum_id: spec for spec in spectra}
-
-    # add edges info to the spectra
-    load_edges(edges_file, spec_dict)
-
-    gnps_format = identify_gnps_format(nodes_file, quant_table_file
-                                       is not None)
+def load_gnps(strains, nodes_file, quant_table_file, metadata_table_file,
```

Comment:
The function name and return values are different with previous function, please keep them consistent